scripts/install_dependencies.py

```diff
@@ -41,24 +41,24 @@ def git_clone(repo_url, branch, dst, recursive=False):
     # Clone the repo
     returncode, stdout, stderr = run_program(
         ['git', 'clone', repo_url, dst] + extra_args)
-    print stdout
+    print(stdout)
```

Comment:
I would print the stdout only in the case the returncode is nonzero, below (same comment on line 56)

Reply:
Same as above, the more information the better.